src/clojisr/v1/r.clj

```diff
@@ -49,7 +51,11 @@
 (def clj->java->r (comp java->r clj->java))
 (def clj->r clj->java->r)
 
-(defn r->java->clj [r-object] (-> r-object r r->java java2clj/java->clj))
+(defn r->java->clj [r-object & options] 
+  (-> r-object 
+      (r options) 
```

Comment:
Why `r` should know about options?

Reply:
hmm. Not sure any more. Was some time ago.
I think the idea was to "be able" to pass options through all layers.

Reply:
Options should be passed only to the `java->clj` function.

Multidimensional conversion is done here: https://github.com/scicloj/clojisr/blob/master/src/clojisr/v1/impl/java_to_clj.clj#L94-L100